cpg_workflows/inputs.py

```diff
@@ -105,7 +105,7 @@ def _populate_cohort(cohort: Cohort, sgs_by_dataset_for_cohort, read_pedigree: b
     TODO (mwelland): so we can lose a layer of the structure here
     """
     for dataset_name in sgs_by_dataset_for_cohort.keys():
-        dataset = cohort.create_dataset(dataset_name)
+        dataset = cohort.create_dataset(dataset_name[:-5] if dataset_name.endswith('-test') else dataset_name)
```

Comment:
I reckon clearer just as

```suggestion
        dataset = cohort.create_dataset(dataset_name.removesuffix('-test'))
```